components/shared/Breadcrumbs/Breadcrumbs.module.css

```
@@ -53,7 +52,7 @@
     top: 0;
     padding: 0 0.7rem;
 
-    @media (min-width: $smallRem) {
+    @media (min-width: 40em) {
     }
```

Stylelint (17.13.0)

[error] Empty block (block-no-empty)

(block-no-empty)